src/OpenTelemetry.Instrumentation.Process/ProcessInstrumentationOptions.cs

```diff
@@ -21,4 +21,9 @@ namespace OpenTelemetry.Instrumentation.Process;
 /// </summary>
 public class ProcessInstrumentationOptions
 {
+    /// <summary>
+    /// Gets or sets the flag indicating whether CPU time should be further broken down by its states.
+    /// The CPU state could be one of the following type: system, user, or wait.
+    /// </summary>
+    public bool? CpuStatesEnabled { get; set; } = false;
```

Comment:
Just curious, why is this a `Nullable<bool>`? Is `null` state meaningful?

nit:

```csharp
    /// <summary>
    /// Gets or sets a value indicating whether or not CPU time should be further broken down by its states.
    /// </summary>
   /// <remarks>The CPU state could be one of the following types: system, user, or wait.</remarks>
```

Reply:
Yeah... I agree that Null state is not really meaningful here; removed.
And I've updated the XML as suggested. Thanks!